components/TextFields/src/ContainedInputView/private/MDCTextControlStyleFilled.m

```diff
@@ -0,0 +1,442 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "MDCTextControlStyleFilled.h"
+
+#import <Foundation/Foundation.h>
+
+#import "MDCTextControl.h"
+#import "MDCTextControlVerticalPositioningReferenceFilled.h"
+#import "UIBezierPath+MDCTextControlStyle.h"
+
+static const CGFloat kFilledContainerStyleTopCornerRadius = (CGFloat)4.0;
+static const CGFloat kFilledContainerStyleUnderlineWidthThin = (CGFloat)1.0;
+static const CGFloat kFilledContainerStyleUnderlineWidthThick = (CGFloat)2.0;
+
+static const CGFloat kFilledFloatingLabelScaleFactor = 0.75;
+
+@interface MDCTextControlStyleFilled () <CAAnimationDelegate>
+
+@property(strong, nonatomic) CAShapeLayer *filledSublayer;
```

Comment:
It might be worth commenting/ pragma mark grouping the ones required for MDCTextControl protocol.

Reply:
actually i missunderstood these never mind